src/main/java/github/scarsz/discordsrv/util/PlayerUtil.java

```diff
@@ -116,6 +116,8 @@ public static void notifyPlayersOfMentions(Predicate<? super Player> predicate,
                         })
                         .collect(Collectors.toList());
 
+        if (!DiscordSRV.config().getBoolean("MinecraftMentionSound")) return;
```

Comment:
Should short circuit higher in the method, after the predicate null check preferably